src/main/scala/temple/utils/FileUtils.scala

```diff
@@ -105,4 +112,36 @@ object FileUtils {
     )
     foundFiles
   }
+
+  /**
+    * Build a sequence of filenames of files present in the provided filepath
+    * @param path The root path to explore
+    * @param cwd The location to give paths relative to
+    * @return A sequence of filenames, relative to the provided path
+    */
+  private def buildFilenameMap(path: Path, cwd: Path): Seq[File] = {
+    var foundFiles: Seq[File] = Seq.empty
```

Comment:
I realise it’s the same for the map, but this would be better served by a val of type `mutable.Buffer`, to avoid all the list copying.